rsrc/rsrc.go

```diff
@@ -0,0 +1,107 @@
+package rsrc
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/akavel/rsrc/binutil"
+	"github.com/akavel/rsrc/coff"
+	"github.com/akavel/rsrc/ico"
+	"github.com/akavel/rsrc/internal"
+)
+
+const (
+	RT_ICON       = coff.RT_ICON
+	RT_GROUP_ICON = coff.RT_GROUP_ICON
+	RT_MANIFEST   = coff.RT_MANIFEST
+)
+
+// on storing icons, see: http://blogs.msdn.com/b/oldnewthing/archive/2012/07/20/10331787.aspx
+type GRPICONDIR struct {
```

Comment:
Uh, oh, this type now becomes exported (previously was hidden in package main); I don't want it to litter the public API of the package; please prefix this and GRPICONDIRENTRY with `_` and check if it still compiles. If not, move it to internal. Also please delete the helper const block above and fix the code to just use appropriate consts from pkg coff.